source/extensions/filters/listener/http_inspector/http_inspector.cc

```diff
@@ -104,9 +104,9 @@ ParseState Filter::onRead() {
   const Api::SysCallSizeResult result =
       os_syscalls.recv(socket.ioHandle().fd(), buf_, Config::MAX_INSPECT_SIZE, MSG_PEEK);
   ENVOY_LOG(trace, "http inspector: recv: {}", result.rc_);
-  if (result.rc_ == -1 && result.errno_ == EAGAIN) {
+  if (SOCKET_FAILURE(result.rc_) && result.errno_ == SOCKET_ERROR_AGAIN) {
```

Comment:
out of curiosity, do we have any more places with the old-style comparison that should be converted to SOCKET_FAILURE?

Reply:
I found a few more and converted them, it was tough to think of a format or other automation to detect it as it is dependent on if the return code comes from a socket function, lmk if you think there is a good way to auto detect this